mama/c_cpp/src/c/mama.c

```diff
@@ -235,6 +242,16 @@ mama_loadPayloadBridgeInternal  (mamaPayloadBridge* impl,
 mama_status
 mama_loadEntitlementBridgeInternal  (const char* name);
 
+void MAMACALLTYPE mamaImpl_entitlementDisconnectCallback (
```

Comment:
should we move this to `entitlementinternal.h` (redefined identically in mama.c)?